CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ The puppet agent software must be installed on a Cisco Nexus platform in the `Gu
 ### Added
 * Extend syslog_server with attribute:
    * `facility`
+* Added `banner` with attributes:
+   * `motd`
+
+*note: due to bug in NXAPI multiline banners cannot be set in NXOS prior to `7.0(3)I7.4` / `9.2(1)`*
```

Comment:
We should also indicate that multiline banners are only supported on `n3k` and `n9k` platforms running these versions.